src/Tag/test/TagSpec.tsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
 import ReactTestUtils from 'react-dom/test-utils';
-import { getDOMNode } from '@test/testUtils';
 import { testStandardProps } from '@test/commonCases';
 import Tag from '../Tag';
 import Sinon from 'sinon';
+import { render } from '@testing-library/react';
 
 describe('Tag', () => {
   testStandardProps(<Tag />);
+
   it('Should output a Tag', () => {
-    const instance = getDOMNode(<Tag />);
-    assert.equal(instance.className, 'rs-tag rs-tag-md rs-tag-default');
+    const { container } = render(<Tag />);
+    expect(container.firstChild).to.have.class('rs-tag');
```

Comment:
Please don't change the purpose of the old test. The updated test should still be checking `rs-tag rs-tag-md rs-tag-default` classes.